### PR TITLE
[site] Correct site usage of fluido so it inherits version from pom

### PR DESCRIPTION
**Pull request by hazendaz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -404,8 +405,7 @@
             <plugin>
                 <groupId>org.apache.maven.skins</groupId>
                 <artifactId>maven-fluido-skin</artifactId>
-                <!-- Must ALSO update this version in src/site/site.xml file as that file doesn't inherit from this pom! -->
-                <version>1.11.1</version>
+                <version>${fluido.version}</version>
```
Comment (michael-o) [inline pom.xml:408]: This entire block looks totally wrong. The skin is not a plugin. What is the purpose here?

**Comment by hazendaz** [comment]
Overlooked that myself.  It should only be a dependency to site plugin so versions plugin reports on it.  If not using that certainly can go from there.  But may also be needed to trigger renovate or dependabot if version tracked in pom.  Not certain on that.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by hazendaz** [comment]
I will add onto this PR to fix misconfiguration as well.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by hazendaz** [comment]
switched fluido skin to dependency.  project does have both versions plugin and dependabot.

**Comment by hazendaz** [comment]
not sure what the failure test is from as added change was just to move plugin into dependency.

**Comment by davewichers** [comment]
When I run: mvn clean test site, using this, I get this error:

...
[INFO] Rendering site with default locale English (en)
[INFO] Relativizing decoration links with respect to localized project URL: https://github.com/nahsra/antisamy
[INFO] Rendering content with org.apache.maven.skins:maven-fluido-skin:jar:1.11.2 skin.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  18.633 s
[INFO] Finished at: 2023-01-24T12:57:13-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:4.0.0-M3:site (default-site) on project antisamy: RendererException while preparing context for skin: Cannot use skin: has [1.11.1,2.0.0-M1) Doxia Sitetools prerequisite, but current is 2.0.0-M3 -> [Help 1]

**Comment by michael-o** [comment]
> When I run: mvn clean test site, using this, I get this error:
> 
> ... [INFO] Rendering site with default locale English (en) [INFO] Relativizing decoration links with respect to localized project URL: https://github.com/nahsra/antisamy [INFO] Rendering content with org.apache.maven.skins:maven-fluido-skin:jar:1.11.2 skin. [INFO] ------------------------------------------------------------------------ [INFO] BUILD FAILURE [INFO] ------------------------------------------------------------------------ [INFO] Total time: 18.633 s [INFO] Finished at: 2023-01-24T12:57:13-05:00 [INFO] ------------------------------------------------------------------------ [ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:4.0.0-M3:site (default-site) on project antisamy: RendererException while preparing context for skin: Cannot use skin: has [1.11.1,2.0.0-M1) Doxia Sitetools prerequisite, but current is 2.0.0-M3 -> [Help 1]
> 
> FYI to @michael-o

This is intentional and expected. You need to upgrade both plugin and skin.

**Comment by davewichers** [comment]
I was thinking that might be the case. As such, I think I can merge this in without upgrading the version of Fluido, because the fixed versions of the rest of the plugins aren't available yet. These changes will at least cause the pom config to drive the version of fluido being used in the site.xml file. Once the 3-4 plugins I'm using support what is needed/required by maven-site-plugin:4.0.0-M4, then I can upgrade all of them at once.

**Comment by davewichers** [comment]
@hazendaz - Thanks for contributing this fix to my pom and site.xml file.

**Pull request merged**